### PR TITLE
feature: ELY-151 add users and groups members to managed GSuite Groups

### PR DESCRIPTION
**Pull request by sclausson**

### CODE REVIEW

File: modules/group/main.tf [hunk 1]
```diff
@@ -9,3 +9,26 @@ resource "gsuite_group" "local" {
   name        = var.name
   description = "Created by Terraform"
 }
+
+resource "gsuite_group_members" "local_users" {
+  count = var.name != var.mock_name && (data.gsuite_group.local.name == null || data.gsuite_group.local.description == "Created by Terraform") ? 1 : 0
+  group_email = "${gsuite_group.local[0].email}"
+
+  dynamic "member" {
```
Comment (sclausson) [inline modules/group/main.tf]: I couldn't figure out how to NOT repeat these two dynamic blocks -- once for Groups and a second time for Users. Do you know @akovtun-extenda ?

Comment (anakovt) [reply modules/group/main.tf]: no, I'm not sure how we can change it. I just went through a couple of examples and saw repeating of dynamic blocks.

File: modules/group/main.tf [hunk 2]
```diff
@@ -9,3 +9,26 @@ resource "gsuite_group" "local" {
...
+    for_each = var.members.users
+
+    content {
+      email = member.value.email
+      role  = "MEMBER"
+    }
+  }
+}
```
Comment (anakovt) [inline modules/group/main.tf]: Add missing LF please

File: vars.tf [hunk 3]
```diff
@@ -36,3 +36,21 @@ variable mock_gsuite_group_name {
...
+    users = list(object(
+      {
+        name  = string
+        email = string
+      }
+    ))
+  })
+  description = "Users and Groups to add as GSuite Tribe or Clan Members"
+}
+
```
Comment (anakovt) [inline vars.tf]: Remove this new line here, please

File: vars.tf [hunk 4]
```diff
@@ -36,3 +36,21 @@ variable mock_gsuite_group_name {
   default     = "placeholder-123"
   description = "Due limitations with Terraform Count and data resource lookups we must use a mock email address instead of an empty value."
 }
+
+variable "gsuite_group_members" {
```
Comment (anakovt) [inline vars.tf]: remove the quotes around the name as you did above

File: modules/group/vars.tf [hunk 5]
```diff
@@ -12,3 +12,20 @@ variable email {
   type        = string
   description = "Email address of GSuite Group"
 }
+
+variable "members" {
```
Comment (anakovt) [inline modules/group/vars.tf]: remove the quotes around the name (or is it something specific in v12 for type = object?)

**Pull request merged**